.gitignore

```diff
@@ -8,3 +8,4 @@
 # Wasm filter
 /wasm_filter/target
 
+/compose/control-plane/static/*
```

Comment:
Note that you are ignoring the contents of this directory _and adding_ the binary wasm filter in this same commit. The binary should be removed.